tests/bson/bson-value-fromPHP-001.phpt

```diff
@@ -0,0 +1,181 @@
+--TEST--
+MongoDB\BSON\Value::fromPHP()
```

Comment:
I've only created a single test for rapid prototyping. This will be split up into multiple tests in the future.